### ISSUE TITLE
Sprint 17.02.08 - Committed

### ISSUE THREAD

**Issue by rfrandse** [openbmc/openbmc#1080]
-------------------------------------------------------------------------------------------------
**Andrew:geissonator -** 

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1094 (final state removal)
    - 02/14: Initial code written to remove old state from obmcutil
    - 02/16: Got previous code merged, working lots of other issue
    - 02/17: Typed up initial email, need to send and start coding
    - 02/20: Worked a lot on openFSI, talked with Brad on some refactoring in this area, have initial note typed up.
    - 02/21: Email sent, code up again for obmcutil changes.  Need to finish up removal from other areas.
-------------------------------------------------------------------------------------------------
**Josh:JoshDKing -** 
**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1070 (REST reboot hangs BMC)
- https://github.com/openbmc/openbmc/issues/1072 (core dump of bmc state service)
- Fix up standby target retrieval
    - 02/21: Finishing this up today
-------------------------------------------------------------------------------------------------
**Matt B.:msbarth -** 
**17.01.11:**
- https://github.com/openbmc/openbmc/issues/959 (fan presence detection)
    - 02/09: Reorganizing to hook into new inventory code
    - 02/10: More code written yesterday, working on x86 SDK, enabled some more CI yesterday
    - 02/13: Commits have been re-done, x86 SDK is looking good now
    - 02/14: Pushed up review commit to own fork, to get everything to Brad for review.
    - 02/15: Doing inventory changes, questions with Brad/Patrick after today
    - 02/16: Code reviews, updates for fan presence after Brad/Patrick discussion
    - 02/20: Code back up for review, inventory commit next (required code merged)
    - 02/21: Most code up, needs error logging

- Misc
    - 02/21: Looked into Vishwa gtest issue, LD_CONFIG problem, run after everything installed

**17.02.08:**
- carry over
-------------------------------------------------------------------------------------------------
**Ratan:ratagupt -**
**17.02.08:**
- https://github.com/openbmc/openbmc/issues/781 (functional IPMI state to inventory mapping)
    - 02/10: Started looking into it, need some discussion with Brad
    - 02/14: Started working this as well
    - 02/21: Done patching data from MRW, working the code, need recipe change

- https://github.com/openbmc/openbmc/issues/1091 (ipmi-fru-parser map chassis FRU data)
    - 02/13: Started work, will push up initial code tomorrow
    - 02/14: Will upload code today
    - 02/15: Code up, Deepak and Spinler to review
    - 02/16: Tested with latest build, looking good, more reviews from Deepak/Spinler.
    - 02/17: Need re-review from Spiner and Deepak
    - 02/21: Ready for Patrick, need priority 
-------------------------------------------------------------------------------------------------
**Tom:tomjoseph83 -**
**17.01.11:**
- https://github.com/openbmc/openbmc/issues/870 (encrypted rmcp, part 2)
    - 02/02: Waiting on Patrick review
    - 02/10: Needed to rebase these changes, ran into some merge issues, got everything through, ready for Patrick again.
    - 02/14: Patrick needs to look at this.
    - 02/15: Patrick needs to look at this.
    - 02/16: Patrick merged 3 of the patches, 13 more pending, Tom addressed comments today and pushed up.
    - 02/17: Nothing happened yesterday, Deepak reviewed today, had comment, Tom to address.  Rest of patches ready for Patrick.
    - 02/20: No progress, most patches ready for Patrick
    - 02/21: Ready for Patrick

**17.01.25:**
- https://github.com/openbmc/openbmc/issues/856 (Attach RMCP session to SOL, part 1)
    - 02/09: Header files up for review, Deepak
    - 02/14: Continued work on this, need Deepak review
    - 02/15: Need Deepak review for the headers, continuing implementation
    - 02/17: Still working through
    - 02/20: Code up, getting close
    - 02/21: Probably going into next sprint

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1110 (network ipmi not working)
    - 02/10: New issue, need Brad input
    - 02/14: Need some thoughts from Brad.
    - 02/15: Discussed with Brad, got some thoughts, delay until first IPMI command comes in

- https://github.com/openbmc/openbmc/issues/1150 (boot_flags not working)
    - 02/17: Need to get this issue over to petitboot

- https://github.com/openbmc/openbmc/issues/1145 (dbus resource or busy)
    - 02/17: Need a plan on someone to look into this
-------------------------------------------------------------------------------------------------
**Deepak:** 
**17.01.25:**
- Overall mfg stage 2 interface to tools team
    - 02/09: Got on system with host ipmi enabled, got partial inventory creation before seg fault, found issues, working on fixes.  FRU ID 0 fix, Brad PIM fix, then lab went down.
    - 02/10: Got full inventory working on witherspoon!
    - 02/13: Built image over weekend with Brad's branch + some changes from Deepak.  Could not get system booting.  Fell back to latest build and loaded on patches.
    - 02/15: Did inventory bringup on w37 with new PNOR from Hostboot, everything looking good.  Deepak pinged remaining patches we need merged.
- https://github.com/openbmc/openbmc/issues/1028 (server process callout design)
    - 02/09: Callout README update, just needs review and done
    - 02/15: +1 on README from Andrew, need to get through Patrick today
    - 02/16: Tested latest build, looking good!  Did some code reviews
    - 02/21: Discussion with Ratan on design and such

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1092 (mrw sysfs to inventory tool)
- https://github.com/openbmc/openbmc/issues/1098 (ipmi - fieldReplaceable and cacheable)
- https://github.com/openbmc/openbmc/issues/1099 (openpower - fieldReplaceable and cacheable)
    - 02/16: Started looking, will finish up 1098 1099 first.
    - 02/20: All code up for review
    - 02/21: Updated code
-------------------------------------------------------------------------------------------------
**Vishwa:vishwabmc -**
**17.01.11:**
- https://github.com/openbmc/openbmc/issues/802 (move all existing code to new LED interfaces)
    - 02/13: Made some code changes over the weekend, need Matt S. review.
    - 02/15: Have +1's, need to change names?
    - 02/17: Sent out email to mailing list
    - 02/21: Worked on this today, looked at design, talked with Nan.  Code up for review.

- https://github.com/openbmc/openbmc/issues/930 (soft power off in host ipmi with watchdog timers)
    - https://gerrit.openbmc-project.xyz/#/c/1972/
    - 02/10: Reworked code based on review comments, one open question in review on common code
    - 02/13: Most code has +1's, need a few more and then ready for Patrick
    - 02/15: Addressed review comments, continue to work with Barth
    - 02/17: Worked on makefile and configure.ac stuff, got CI passing, has gtest issue
    - 02/21: Fixed some review comments, had tech discussion on service returns, few more things to fix

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/766 (host checkstop refactor)
    - 02/20: 

- Defect Screener
-------------------------------------------------------------------------------------------------
**Eddie:eddiejames -**
**16.12.28:**
- https://github.com/openbmc/openbmc/issues/624 (initial occ intf refactor)
- https://github.com/openbmc/openbmc/issues/625 (p9 occ data)
    - 02/09: No responses
    - 02/10: Joel sent some feedback, will address those today
    - 02/13: Replied to comments, will address code shortly
    - 02/14: Will work on patch set for this today
    - 02/15: Pushed up some new occ patches to mailing list (version 8)
    - 02/17: Nothing on mailing list
    - 02/20: Nothing on mailing list
    - 02/21: Nothing on mailing list

**17.01.11:**
- FSI based I2C driver (RTC story?)
    - 02/13: Joel built everything into openfsi, #1083 needs to be looked at
    - 02/16: Will fix the increment issue today
    - 02/17: Will send out device number fix today

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1004 (I2C migration to official interfaces)
    - 02/10: Nothing back from Joel so working on SBE FIFO
    - 02/15: Nothing from Joel on base I2C rewrite

- Corruption of 2nd 32 bits
    - 02/20: Will look into this today
    - 02/21: Patch out and merged!
-------------------------------------------------------------------------------------------------
**Chris -**
**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1082 (Verify fsi ops, i2c)
    - Some comments on 1085 and 1086
    - 02/10: All mfg stage 2 changes are in, Chris to ask for us to enable it today
    - 02/13: Joel merged in patches, updated some issues.  #1086 is hub related, not critical.
    - 02/14: Trying to get a 2 chip system with a serial port, need help from Uma to get one.
    - 02/15: Got 2 chip going yesterday, was able to access chip, hitting a few random bus fails, looking into that.  Patches will be sent out today.
    - 02/16: Everything was running pretty well last night, has 2 chip and error recovery.  Will get patches out today.
    - 02/17: Has review comment on hub and error recovery patch.  Seems cosmetic-ish, Chris to work patch comments, will discuss with LTC on Monday.
    - 02/20: Will get v2 out today, will work with Eddie a bit on the SCOM issue
    - 02/21: Nearly finished up v2, will send out today, then back to community work
-------------------------------------------------------------------------------------------------
**Brad:bradbishop -**
**17.01.25:**
- https://github.com/openbmc/openbmc/issues/956 (add object lifecycle yaml for inventory from ipmi)
    - 02/09: Code up, going through review process
    - 02/17: Need review from Deepak
- https://github.com/openbmc/openbmc/issues/995 (create error if BMC EEPROM not found)
    - 02/09: Needs task break down, needs yaml code, lets wait
    - 02/17: Waiting for Adriana code to get in (auto generation of code)
    - 02/21: Other work prioritized ahead of this, will need to move

**17.02.08:**
- Inventory
    - 02/10: Did lots of things to get inventory moving along, going to fix library linking issues.  Will look at 1106 issue and other library linking problems today.
    - 02/13: Got all the package config stuff working on Friday.  Used a new macro that causes CI to fail, working on that this morning.
    - 02/14: Did a lot of reviews, minor fix up's for inventory.  Got the recipe through jenkins last night! (had to take LED stuff out).  hwmon and inventory manager ready!  Reviewed all of Deepak/Ratan code. 
    - 02/15: 50% doing code reviews, 50% working on Patrick inventory manager feedback.
    - 02/16: More reviews, reworked inventory code, got it all merged yesterday
    - 02/17: Started digging into SBE FIFO work, sent out some notes, talked with Joel
    - 02/21: Reviews, then worked on REST api back end work for UI team
-------------------------------------------------------------------------------------------------
**Adriana:anoo1 -**

**17.01.25:**
- https://github.com/openbmc/openbmc/issues/1035 (UBI/UBIFS support in kernel)
    - 02/09: Patch sent to mailing list, added tools to utility
    - 02/14: Had discussion in last nights call, need to talk with Patrick a bit
    - 02/17: New patch sent out, feedback provided
    - 02/20: Need new patch sent out
- https://github.com/openbmc/openbmc/issues/1036 (Verify UBI volumes can be created)
    - 02/16: Need to move these stories

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1079 (program to create and update UBI volumes)
    - 02/09: Needs task break down, will do in a couple days

- Misc error log updates
    - 02/20: Got some changes up required by Brad for generating errors
-------------------------------------------------------------------------------------------------
**Michael:mtritz -**
**17.01.25:**
- https://github.com/openbmc/openbmc/issues/942 (disable automatic reboots of host)
    - 02/09: Testing and debugging yesterday on state code.  
    - 02/10: Ran into SDK sdbusplus issues, talked with Patrick, moved to map box calls, fixed it for now.
    - 02/13: Got state manager code working with quiesce, working on host watchdog code now
    - 02/14: Code done, looking for reviews
    - 02/15: Still looking for reviews, Andrew + Vishwa
    - 02/16: Spent a lot of time yesterday reworking code review comments, addressed and ready for re-review.
    - 02/17: More review comments back and forth, needs one more re-review from Andrew
    - 02/20: Working Patrick comment
    - 02/21: Code merged, but we have patch issue

**17.02.08:**
- https://github.com/openbmc/openbmc-test-automation/issues/205 (error log inject tool)
    - 02/09: No work yet, focused on 942
    - 02/15: Looking into background, tested out a few fixes
    - 02/16: Andrew/Michael talked, up next
    - 02/17: Started working on the code
    - 02/20: First tool available, need to discuss with Andrew
    - 02/21: Tools pretty close to what we want, couple tweaks coming
-------------------------------------------------------------------------------------------------
**Gunnar:gtmills**
**17.01.25:**
- https://github.com/openbmc/openbmc/issues/1037
    - 02/10: Addressed Adriana code review comments, need Barth/Adriana re-review
    - 02/13: Some comments from Matt, will look at.
    - 02/14: Need Adriana to take a look
    - 02/15: Adriana to look
    - 02/16: Time for Patrick
    - 02/20: Recipe up, ready for reviews
    - 02/21: Code ready for Patrick, recipe ready for Brad

**17.02.08:**
- https://github.com/openbmc/openbmc/issues/1047 (Parse PNOR TOC and create metadata file)
    - 02/09: Need task list
    - 02/21: Code on it's way
-------------------------------------------------------------------------------------------------
**Matt:spinler -** 
**17.01.11:**
- https://github.com/openbmc/openbmc/issues/629 (integrate device tree into kernel)
    - 02/09: All code up in gerrit, need Andrew/Gunnar to re-review a few
    - 02/10: Everything is +1'd for device scripts, ready for Patrick.  Recipe to copy into kernel ready for Brad/Patrick.  FSI stuff coming, will need updates.
    - 02/13: Worked on getting openFSI stuff into device tree
    - 02/15: Need Patrick merge for perl updates, will hold off on merging actual auto generated device tree until we get through mfg.
    - 02/16: Move this, waiting for mfg

**17.02.08:**
https://github.com/openbmc/openbmc/issues/866 (use openFSI directly to start P8/P9)
    - 02/13: Started looking, need discussion in design meeting.  Port FAPI2 to openBMC?
    - 02/15: Sick yesterday, needs new repo, will use Deepak code to handle FSI errors
    - 02/16: Started working on code to use sysfs to do the FSI calls
    - 02/20: Got initial code ready, need repo from Patrick

https://github.com/openbmc/openbmc/issues/1139 (openFSI bringup fixes)
    - 02/17: Has initial patch ready
    - 02/20: Patches are up and reviewed

- Need support for hostboot commands to set watchdog timeout, and do soft power off's
    - 02/17: Matt to open issues
    - 02/20: Matt found support in our code, so seems like it's working

- Sensor updates
    - 02/17: Trying to get boot progress and boot count going
    - 02/20: Matt to take a look
    - 02/21: Code merged
-------------------------------------------------------------------------------------------------
**Saqib: -** 
- https://github.com/openbmc/openbmc/issues/938 (log.hpp vs phosphor-logging/log.hpp include)
    - 02/09: Found missing config file, will work on it.
    - 02/10: vacation
    - 02/13: Working this today
    - 02/14: Testing the changes, hopefully push up code today
    - 02/15: Need to get Adriana code through first, then we'll do Saqib
    - 02/20: Initial code merged, will now mover over other resos
    - 02/21: Code up for this, needs reviews

- https://github.com/openbmc/openbmc/issues/1136 (quiesce state in host object)
    - 02/16: Will start looking into this
    - 02/20: Code under review
    - 02/21: Ready for re-review

- https://github.com/openbmc/openbmc/issues/1143 (service work)
   - 02/20: Looking into
-------------------------------------------------------------------------------------------------
**Leonel: -** 
**17.02.08:**
- https://github.com/openbmc/openbmc/issues/859 (/run 100% full after reboot)
    - 02/13: Started looking into this one
    - 02/14: System crashed the other day, working through issues
    - 02/15: Met with Angel, looked for RTC battery replacement, can't seem to get system in usable state.
    - 02/16: Working on making the new image for BarE01 that will make that particular log message debug
    - 02/17: Gave Angel new image
    - 02/20: Issue fixed by fixing the battery, we should get system test.  Old systems had SEL when battery got low or had issues.

- https://github.com/openbmc/openbmc/issues/1137 (dbus msg support in openbmc CI)
    - 02/17: Ping Matt Barth for overview
    - 02/21: Having issues getting docker to work with dbus, talk with Patrick
-------------------------------------------------------------------------------------------------
**Patrick:williamspatrick -**
**16.12.28**
- https://github.com/openbmc/openbmc/issues/477 (format to document org.openbmc interfaces)
    - 01/27: Non critical
    
**17.01.11:**
- https://github.com/openbmc/openbmc/issues/851 (client proxy bindings) and https://github.com/openbmc/openbmc/issues/970

**17.01.25:**

**17.02.08:**
- Overall
    - Reviewed a lot of code and merged
    - 02/13: Did a lot of code reviews, next on to Tom and Vishwa code.
    - 02/14: Reviews all day
    - 02/15: Reviews and merging all day.  Looking into the string/char* bug.
    - 02/16: Lots of code reviews, lots of merges, fixed a sdbusplus bug
    - 02/17: Fixed dbus binding issue, did code reviews
-------------------------------------------------------------------------------------------------
**Nan:williamli80 -**
**17.01.11:**
- https://github.com/openbmc/openbmc/issues/770 (pciedetect refactor)
    - 02/13: Has new patch up for 770, needs review from team, no time currently though from team with mfg stage 2 work taking priority.

**17.02.08:**
- carry over
-------------------------------------------------------------------------------------------------
**Yi:adamliyi -**
**16.12.28:**
- https://github.com/openbmc/openbmc/issues/767 (host watchdog refactor)
    - https://gerrit.openbmc-project.xyz/#/c/1513/

**17.01.11:**
- https://github.com/openbmc/openbmc/issues/665 (FRU vpd in inventory)

**17.02.08:**
- carry over
-------------------------------------------------------------------------------------------------
**Lei:mine260309 -** 
**17.01.25:**
- Assist Nan/Yi

**17.02.08:**
- Assist Nan/Yi
